Formula/crun.rb

```diff
@@ -0,0 +1,28 @@
+class Crun < Formula
+  desc "Fast and lightweight fully featured OCI runtime and C library"
+  homepage "https://github.com/containers/crun"
+  url "https://github.com/containers/crun/releases/download/0.20.1/crun-0.20.1.tar.xz"
+  sha256 "d118b9749e0ea90de1bd8bb1314f319f43559bbd275cd9331f2d96cfd1ccf9e0"
+  license any_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
+
+  depends_on "go-md2man" => :build
+  depends_on "libtool" => :build
+  depends_on "python@3.9" => :build
+
+  depends_on "libcap"
+  depends_on "libseccomp"
+  depends_on :linux
+  depends_on "systemd"
+  depends_on "yajl"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "crun", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Thanks for the review. Crun is a container runtime for upper level software to run containers, in this case, substantial tests will involve other packages (e.g., docker, podman). It might not be a good idea to include those packages for testing. Actually even for the crun repository itself check the installation using `crun --version` (see https://github.com/containers/crun#nix). So that I can do is to add some checking to the output of `crun --version`.

Reply:
Why not something like

> If the software cannot function without credentials or requires a virtual machine, docker instance, etc. to run, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.

as suggested in the docs?

We just need something more than checking `--version`.

Reply:
Thanks for the suggestion. I changed it to invoking `crun --rootless=true list -q` and make sure it outputs nothing.